2007-07-23  Mike Frysinger  <vapier@gentoo.org>

	* iconv/Makefile (install-others): Don't define if not build-programs.
	* login/Makefile (install-others): Likewise.
	* posix/Makefile (install-others): Likewise.

--- iconv/Makefile
+++ iconv/Makefile
@@ -56,8 +56,10 @@
 		  iconvconfig.h
 
 others		= iconv_prog iconvconfig
-install-others	= $(inst_bindir)/iconv
 install-sbin	= iconvconfig
+ifeq ($(build-programs),yes)
+install-others	= $(inst_bindir)/iconv
+endif
 
 CFLAGS-gconv_cache.c += -DGCONV_DIR='"$(gconvdir)"'
 CFLAGS-gconv_conf.c = -DGCONV_PATH='"$(gconvdir)"'
--- login/Makefile
+++ login/Makefile
@@ -30,7 +30,9 @@
 CFLAGS-grantpt.c = -DLIBEXECDIR='"$(libexecdir)"'
 
 others = utmpdump pt_chown
+ifeq ($(build-programs),yes)
 install-others = $(inst_libexecdir)/pt_chown
+endif
 
 distribute := utmp-private.h utmp-equal.h pty-private.h
 
--- posix/Makefile
+++ posix/Makefile
@@ -98,7 +98,9 @@
 endif
 others		:= getconf
 install-bin	:= getconf
+ifeq ($(build-programs),yes)
 install-others	:= $(inst_libexecdir)/getconf
+endif
 
 before-compile	:= testcases.h ptestcases.h
 
